src/invidious/helpers/utils.cr

```diff
@@ -365,3 +365,14 @@ def fetch_random_instance
 
   return filtered_instance_list.sample(1)[0]
 end
+
+def reduce_uri(uri : URI | String, max_length : Int32? = 50, suffix : String? = "...") : String
```

Comment:
No need of making `max_length` and `suffix` nillable. Assigning a default value is enough to make the parameters optional. And that can let you avoid those useless `.nil?` checks ^^

Also, please use the horizontal ellipsis unicode character `…` (U+2026) instead of three dots.

Reply:
> No need of making `max_length` and `suffix` nillable. Assigning a default value is enough to make the parameters optional. And that can let you avoid those useless `.nil?` checks ^^

Here the possibility to put `nil` is an artifact of my local version with the configuration [That I had mentioned [here](https://github.com/iv-org/invidious/issues/2218#issuecomment-1050322509)] (Where the default value was taken from the config), I remove that immediately and yes the check `nil?` is obsolete now

> Also, please use the horizontal ellipsis unicode character … (U+2026) instead of three dots.

Indeed, I do it right now